frontend/appflowy_flutter/assets/translations/en.json

```diff
@@ -234,6 +234,15 @@
       "icon": "Icon",
       "selectAnIcon": "Select an icon",
       "pleaseInputYourOpenAIKey": "please input your OpenAI key"
+    },
+    "shortcuts": {
+      "shortcutsLabel": "Shortcuts",
+      "command": "Command",
+      "keyBinding": "Keybinding",
+      "addNewCommand": "Add New Command",
+      "updateShortcutStep": "Press desired key combination and press ENTER",
+      "shortcutIsAlreadyUsed": "This shortcut is already used for - ",
```

Comment:
Maybe `This shortcut is already used for: {}`.

Reply:
Is that the standard to substitute values within the translated strings? TIL :)

Reply:
According to easy_localization yeah